platform/fabric/sdk/dig/sdk.go

```diff
@@ -190,17 +190,17 @@ func registerRWSetLoaderHandlerProviders(in struct {
 	for _, network := range in.CoreConfig.Names() {
 		fsn, err := in.FSNProvider.FabricNetworkService(network)
 		if err != nil {
-			return fmt.Errorf("could not find network service for %s", network)
+			return fmt.Errorf("could not find network service for %s: %w", network, err)
```

Comment:
Please, Arne, given that you are changing this, could replace `fmt` with the package `github.com/hyperledger-labs/fabric-smart-client/pkg/utils/errors`. Thanks.